### PR TITLE
Jakarta Activation 1.2

### PR DESCRIPTION
**Pull request by bshannon**
## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat-spec-x.y.pdf
- [x] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on
      a Compatible Implementation and TCK, to ensure that the spec is
      implementable and testable.
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://oss.sonatype.org/content/repositories/staging/jakarta/activation/jakarta.activation-api/1.2.2/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jaf/tck/eftl/jakarta-activation-tck-1.2.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/jaf/issues/29

Include the following in PR#2:
- [ ] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

**Comment by kwsutter** [comment]
Just putting the "hold" tag on this one to be consistent with the others in this same class -- Java SE 11 drops. We're making some good progress on the Platform project to understand where we go with these in Jakarta EE. Thanks for your patience.

File: activation/1.2/_index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: "Jakarta Activation 1.2"
+date: 2019-10-01
+summary: "Initial release of Jakarta Activation"
+---
+Jakarta Activation defines a set of standard services to: determine the
+MIME type of an arbitrary piece of data; encapsulate access to it;
+discover the operations available on it; and instantiate the
+appropriate bean to perform the operation(s).
+
+* [Jakarta Activation 1.2 Specification Document](./activation-spec_1.2.pdf) (PDF)
```
Comment (kwsutter) [inline activation/1.2/_index.md]: The actual files have the correct names, but these references do not.  There should be a dash "spec-1.2", not an underscore "spec_1.2".

File: activation/1.2/_index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* [Jakarta Activation 1.2 Specification Document](./activation-spec_1.2.html) (HTML)
```
Comment (kwsutter) [inline activation/1.2/_index.md]: Same thing here..

File: activation/1.2/_index.md (hunk 1 above)
Comment (bshannon) [reply activation/1.2/_index.md]: Fixed.

File: activation/1.2/_index.md (hunk 2 above)
Comment (bshannon) [reply activation/1.2/_index.md]: Fixed.

**Comment by kwsutter** [comment]
- [ ] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification and apidocs PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request.
 - [ ] The compatible implementation project/vendor MUST send an email tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as appropriate for the branch management for the project.

**Pull request merged**